Update/rename toolbelt (#189)
vtex/vtex-courses: courses/basic-blocks/steps/00_setup/pt.md

Antes de colocar as mãos na massa e aprender mais sobre o Store Framework no VTEX IO, você precisará definir algumas configurações básicas, como:

- Instalação do **Git**;

> - Instalação da **CLI da VTEX**;

Reviewer: Não seria CLI do VTEX **IO**? (Para acompanhar a versão em inglês)

Author: Verdade! Fiz correndo

Author: ```suggestion
- Instalação da **CLI do VTEX IO**;
```

[https://git-scm.com/downloads](https://git-scm.com/downloads)

> ## Instalando a CLI da VTEX

Author: ```suggestion
## Instalando a CLI do VTEX IO
```

> **A CLI da VTEX** é uma ferramenta VTEX **de linha de comando**. Ele permite que você execute qualquer atividade na plataforma, como criar um novo _workspace_ de desenvolvimento, fazer login em uma conta VTEX, desenvolver novas aplicações ou gerenciar os já existentes, etc.

Author: ```suggestion
**A CLI do VTEX IO** é uma ferramenta VTEX **de linha de comando**. Ele permite que você execute qualquer atividade na plataforma, como criar um novo _workspace_ de desenvolvimento, fazer login em uma conta VTEX, desenvolver novas aplicações ou gerenciar os já existentes, etc.
```

> Uma vez que é a CLI da VTEX que estabelece a comunicação entre o desenvolvedor e a plataforma, você precisará dele para realizar todas as atividades propostas durante esse e os outros cursos.

Author: ```suggestion
Uma vez que é a CLI do VTEX IO que estabelece a comunicação entre o desenvolvedor e a plataforma, você precisará dele para realizar todas as atividades propostas durante esse e os outros cursos.
```

1. Instale o [**Node.js**](https://nodejs.org/). Se você estiver usando um MAC, instale também [**Yarn**](https://yarnpkg.com/);
2. Execute `npm i -g vtex` em seu terminal se estiver usando Windows ou `yarn global add vtex` se estiver usando MAC;

> Você pode executar `vtex` para confirmar se a instalação da CLI da VTEX ocorreu conforme o esperado.

Author: ```suggestion
Você pode executar `vtex` para confirmar se a instalação da CLI do VTEX IO ocorreu conforme o esperado.
```
